scripts/trace-e2e-python.py

```
@@ -0,0 +1,171 @@
+    provider = TracerProvider(resource=Resource.create({"service.name": "awa-python-e2e"}))
+    provider.add_span_processor(BatchSpanProcessor(OTLPSpanExporter(endpoint=OTLP_ENDPOINT, insecure=True)))
+    trace.set_tracer_provider(provider)
+    tracer = trace.get_tracer("trace-e2e")
+
+    # Rust-side pipeline: exports the runtime's job.execute spans.
+    awa.init_telemetry(OTLP_ENDPOINT, "awa-python-e2e-rust", traces=True)
+
+    client = awa.AsyncClient(DATABASE_URL)
+    await client.migrate()
+    tx = await client.transaction()
+    await tx.execute(f"DELETE FROM awa.jobs WHERE queue = '{QUEUE}'")
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)